rsmtool/rsmsummarize.py

```diff
@@ -75,7 +77,15 @@ def check_experiment_dir(experiment_dir, configpath):
                                     "the .json configuration files for rsmtool "
                                     "experiments.".format(full_path_experiment_dir))
 
-        return jsons
+        # Raise an error if the user specified a list of experiment names 
+        # but we found several .jsons in the same directory
+        if experiment_name and len(jsons) > 1:
+            raise ValueError("{} contains several experiment jsons "
+                             "To use custom experiment names "
+                             "You must have a separate "
+                             "directory for each experiment.")
```

Comment:
This format string is missing the argument, right? Is the user supposed to know about experiment JSONs? Perhaps we should say something more general like:

> "{} seems to contain the output of multiple experiments. In order to use custom experiment names, you must have a separate directory for each experiment."